device-registry/src/main/scala/org/genivi/sota/device_registry/DevicesResource.scala

```diff
@@ -197,6 +207,11 @@ class DevicesResource(namespaceExtractor: Directive1[AuthedNamespaceScope],
     } ~
     (path("packages" / "affected") & scope.post) {
       findAffected(authedNs)
+    } ~
+    pathPrefix("package_stats") {
+      (scope.checkReadonly & refinedPackageName & pathEnd) { name =>
```

Comment:
I meant to ask about this. It was like this in resolver, I thought perhaps readOnly enforced get. But I'll change it.